cmd/beacon-chain/powchain/options.go

```diff
@@ -13,16 +19,45 @@ var log = logrus.WithField("prefix", "cmd-powchain")
 func FlagOptions(c *cli.Context) ([]powchain.Option, error) {
 	endpoints := parsePowchainEndpoints(c)
 	executionEndpoint := parseExecutionEndpoint(c)
+	jwtSecret, err := parseJWTSecret(c)
+	if err != nil {
+		return nil, errors.Wrap(err, "could not read JWT secret file for authenticating execution API")
+	}
 	opts := []powchain.Option{
 		powchain.WithHttpEndpoints(endpoints),
 		powchain.WithEth1HeaderRequestLimit(c.Uint64(flags.Eth1HeaderReqLimit.Name)),
 	}
 	if executionEndpoint != "" {
 		opts = append(opts, powchain.WithExecutionEndpoint(executionEndpoint))
 	}
+	if len(jwtSecret) > 0 {
+		opts = append(opts, powchain.WithExecutionClientJWTSecret(jwtSecret))
+	}
 	return opts, nil
 }
 
+func parseJWTSecret(c *cli.Context) ([]byte, error) {
```

Comment:
Cross-check vs the spec here: https://github.com/ethereum/execution-apis/blob/main/src/engine/authentication.md#key-distribution